containers/kubernetes/api-cli/using-load-balancer-annotations.mdx

```diff
@@ -251,7 +251,7 @@ The default value is `10m`. The duration are go's time. Duration (ex: `1s`, `2m`
 ## service.beta.kubernetes.io/scw-loadbalancer-timeout-tunnel
 
 This is the annotation to set the maximum tunnel inactivity time.
-The default value is `10m`. The duration are go's time. Duration (ex: `1s`, `2m`, `4h`, ...).
+The default value is `10s`. The duration are go's time. Duration (ex: `1s`, `2m`, `4h`, ...).
```

Comment:
Same here (https://github.com/scaleway/scaleway-cloud-controller-manager/blob/2f70ff74f90b6b8240da6ded9bf61ae87884974a/scaleway/loadbalancers.go#L1481), the previous value was correct.